spotbugs-annotations/src/main/java/edu/umd/cs/findbugs/annotations/CheckForNull.java

```diff
@@ -33,15 +33,12 @@
  *
  * When this annotation is applied to a method it applies to the method return
  * value.
- *
- * @deprecated - use {@link javax.annotation.CheckForNull} instead.
  **/
 @Documented
 @Target({ ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER, ElementType.LOCAL_VARIABLE })
 @Retention(RetentionPolicy.CLASS)
 @javax.annotation.Nonnull(when = When.MAYBE)
```

Comment:
I think these should probably be removed if removing "JSR-305" stuff

Reply:
It seems really hard to delete dependency on jsr-305, or we cannot support feature depending on `@TypeQualifierDefault` such as `ReturnValuesAreNonnullByDefault`.

Reply:
If you need defaults, you need your own defaults annotation. There is no JSR 305 and this there are no annotations from it *until* the JSR page publishes at least a draft javadocs.